app.json

```diff
@@ -13,76 +13,13 @@
   ],
   "env": {
     "BROKER_URL": {
-      "required": true
-    },
```

Comment:
Why are we removing these requirements?  I'm not saying it's wrong to do so.  I just don't know this area well enough to know why these are bad to have.  If I understand, these will help us ensure we set these required values on a new instance?

Reply:
The original app.json was generated automatically when I enabled the feature Heroku Review Apps. It's only now that I have spent time figurying out which ones we actually do need.

`"required" true` is the same as saying copy the env variable from the app we're basing this off.

In some cases, we don't need them because we can't configure the Heroku scheduler to run the command to update the bugs cache.

In other cases, the env variables get defined when the add-on is added.

In other words, we might be able to use some of these env variables in the future if we invest more but for now this is the minimum amount we need to get the ingestion pipeline working, the backend and the frontend.

@camd does this answer satisfy you?

Reply:
Yep, makes sense.  Thanks for explaining.  :)